packages/backend/src/index.ts

```diff
@@ -28,6 +30,17 @@ const store = new KnexSessionStore({
     sidfieldname: 'sid',
 });
 const app = express();
+Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    integrations: [
+        new Sentry.Integrations.Http({ tracing: true }),
+        new Tracing.Integrations.Express({
+            app,
+        }),
+    ],
+    tracesSampleRate: 1.0,
+});
+app.use(Sentry.Handlers.requestHandler());
```

Comment:
We should specify the user fields (default is ['id', 'username', 'email']) and add the tracking handler.

```
app.use(
    Sentry.Handlers.requestHandler({
        user: [
            'userUuid',
            'firstName',
            'lastName',
            'email',
            'organizationUuid',
            'organizationName',
        ],
    }),
);
app.use(Sentry.Handlers.tracingHandler());
```

Reply:
TY I'll just use the anon fields